README.md

```diff
@@ -97,6 +97,16 @@ Learn more
 ----------
 After you are up and running see the [User Manual](http://fitstool.org/user-manual) for more documentation.
 
+
+Development
+-----------
+
+This project uses a code formatter to ensure consist formatting. To run the formatter:
+
+    mvn spotless:apply
+
+When the project builds, it checks the formatting and will fail if there are any files that are not formatted per the standard.
```

Comment:
Can you reference the style-guide that is now being enforced? It may be frustrating for developers attempting to conform to an unknown standard.

Reply:
Done